config.yaml

```diff
@@ -0,0 +1,7 @@
+receivers:
```

Comment:
Is this just for demo purpose?

Reply:
Yes. I will take out the config file as each developer would have to create their own.